bootstrap/config/config.go

```diff
@@ -866,6 +866,28 @@ func (cp *Processor) listenForCommonChanges(fullServiceConfig interfaces.Configu
 				// envVars override of one of the Writable fields, so on the first update we can just save a copy of the
 				// common writable for comparison for future writable updates.
 				if isFirstUpdate {
+					if raw == nil {
```

Comment:
For code readability, I'd suggest to add some comments here to describe the basic logic to handle nil raw.